### PR TITLE
links for news item tags

### PR DESCRIPTION
**Pull request by jduss4**
closes https://github.com/CDRH/african_poetics/issues/179

### CODE REVIEW

**Comment by jduss4** [comment]
Because the helper / partial I was calling only displays text OR links to individual related pages (like a work, an event, a person, etc) I was unable to use its functionality for the keywords search results.  Therefore, I added the unordered list directly to the news item show page.

File: app/views/inthenewsnewsitems/show.html.erb [hunk 1]
```diff
@@ -86,11 +86,13 @@
...
   <div class="item_data">
-    <%= render partial: "inthenews/simple_association_list", locals: {
-      item: @record,
-      assoc: "tags",
-      path: nil
-    } %>
+    <ul>
+      <% @record.tags.each do |tag| %>
+        <li>
+          <%= link_to tag.name, inthenews_news_items_search_path(f: ["keywords|#{tag.name}"]) %>
```
Comment (sgar7) [inline app/views/inthenewsnewsitems/show.html.erb:92]: Is this where you're adding the unordered list?

**Review by sgar7 — COMMENTED** [review]
Pretty straightforward; just confused if line 92 is adding that unordered list. (Am I reading this correctly?)

File: app/views/inthenewsnewsitems/show.html.erb (hunk 1 above)
Comment (jduss4) [reply app/views/inthenewsnewsitems/show.html.erb:92]: @sgar7 pretty much!

I open the `<ul>` on line 89, then I'm saying "for all the tags related to this database record, add a `<li>` with a link inside of it"  So line 92 is adding a link, rails-style, which will come out to `<a href="the_search_path">Tag Name</a>`

Comment (sgar7) [reply app/views/inthenewsnewsitems/show.html.erb:92]: Got it!

**Pull request merged**